cli/goss.yml

```diff
@@ -0,0 +1,30 @@
+command:
+  composer:
+    exec: "composer"
+    exit-status: 0
+    stdout:
+      - "Available commands"
+
+  composer2:
```

Comment:
I think it's time we make this `composer`.

Reply:
Definitely. Let's do a separate PR.